src/core/index.js

```diff
@@ -136,7 +125,49 @@ class IPFS extends EventEmitter {
       isIPFS: isIPFS
     }
 
-    boot(this)
+    series([
+      (cb) => {
+        waterfall([
+          (done) => this._repo.config.get((error, config) => {
+            if (error) {
+              this.log('Could not load config', error)
```

Comment:
It’s probably not a huge deal since the logs only show if someone’s explicitly set `DEBUG=jsipfs`, but this will log even when things are just fine — e.g. if running `jsipfs init`. It makes it look like something went wrong even when the error is expected and OK.

Would it be better to check whether the repo exists first or to check `this._options.init`? (Are there other situations when this *should* fail?)